test/unit/repoter.js

```diff
@@ -0,0 +1,39 @@
+var reporter = require('../../lib/reporter.js');
```

Comment:
this is a really small, nitpicking issue, but you mis-spelled the name of the file - please fix :-)